client-misk/src/main/kotlin/app/cash/backfila/client/misk/BackfilaModule.kt

```diff
@@ -2,83 +2,57 @@ package app.cash.backfila.client.misk
 
 import app.cash.backfila.client.misk.client.BackfilaClientConfig
 import app.cash.backfila.client.misk.client.BackfilaClientLoggingSetupProvider
+import app.cash.backfila.client.misk.client.BackfilaClientModule
 import app.cash.backfila.client.misk.client.BackfilaClientNoLoggingSetupProvider
+import app.cash.backfila.client.misk.embedded.EmbeddedBackfilaModule
 import app.cash.backfila.client.misk.internal.BackfilaClient
 import app.cash.backfila.client.misk.internal.BackfilaStartupConfigurator
-import app.cash.backfila.client.misk.internal.BackfillOperator
-import app.cash.backfila.client.misk.internal.HibernateBackfillOperator
 import app.cash.backfila.client.misk.internal.RealBackfilaClient
-import com.google.inject.Binder
 import com.google.inject.BindingAnnotation
 import com.google.inject.Provides
-import com.google.inject.TypeLiteral
-import com.google.inject.multibindings.MapBinder
 import com.squareup.moshi.Moshi
 import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
 import javax.inject.Singleton
 import kotlin.reflect.KClass
-import kotlin.reflect.jvm.jvmName
 import misk.ServiceModule
 import misk.inject.KAbstractModule
 
 /**
- * Backfila-using applications install this and either [EmbeddedBackfilaModule] (testing and
- * development) or [BackfilaClientModule] (staging and production).
+ * Backfila-using applications install this, one or more specific client implementation modules such
+ * as [HibernateBackfilaClientModule] and either [EmbeddedBackfilaModule] (testing and development)
+ * or [BackfilaClientModule] (staging and production).
  */
 class BackfilaModule(
   private val config: BackfilaClientConfig,
-  @Deprecated(message = "Multibind backfills instead")
+  @Deprecated(message = "Multibind backfills using 'HibernateBackfillInstallModule.create' instead")
```

Comment:
have you converted off of this? can remove if so

Reply:
Not entirely yet. Maybe I'll just do it. But I'll remove it in a later pr.